File: C++/fibonacci.cpp
#include <iostream>
#include <vector>

int fibonacciSeq(int number) {
    if (number <= 1) {
        return number;
    } else {
        return fibonacciSeq(number - 1) + fibonacciSeq(number - 2);
    }
}

int main() {
    int myInput;
    std::cout << "Enter Your Fibonacci Number: ";
    std::cin >> myInput;

    std::vector<int> myFibonacciList;

    for (int i = 0; i < myInput; i++) {
        myFibonacciList.push_back(fibonacciSeq(i));
    }

    if (myInput <= 0) {
        std::cout << "The Number is Wrong." << std::endl;
    } else {
        std::cout << "Fibonacci Sequence: ";
        for (int num : myFibonacciList) {
            std::cout << num << " ";
        }
        std::cout << std::endl;
    }

    return 0;
}

File: JavaScript/fibonacci.js
function fibonacciSeq(number) {
    if (number <= 1) {
        return number;
    } else {
        return fibonacciSeq(number - 1) + fibonacciSeq(number - 2);
    }
}

const myInput = parseInt(prompt("Enter Your Fibonacci Number: "));
const myFibonacciList = [];

for (let i = 0; i < myInput; i++) {
    myFibonacciList.push(fibonacciSeq(i));
}

if (myInput <= 0) {
    console.log("The Number is Wrong.");
} else {
    console.log(`Fibonacci Sequence: ${myFibonacciList}`);
}